tools/IoTEdgeDevOps/DevOpsLib/BugManagement.cs

```diff
@@ -73,6 +83,12 @@ public async Task<string> CreateBugAsync(string branch, VstsBuild build)
                         rel = "Hyperlink",
                         url = $"{build.WebUri}"
                     }
+                },
+                new
+                {
+                    op = "add",
+                    path = "/fields/System.AssignedTo",
+                    value = bugOwnerEmail
                 }
             };
```

Comment:
Discussed with Damon and I will also add a tag here to specify that the bug is autogenerated from a pipeline failure